shell/platform/common/client_wrapper/include/flutter/encodable_value.h

```diff
@@ -158,6 +159,7 @@ using EncodableValueVariant = std::variant<std::monostate,
 // std::vector<double>  -> Float64List
 // EncodableList        -> List
 // EncodableMap         -> Map
+// std::vector<float>   -> Float32List
```

Comment:
Nit: since this is a comment about mapping, not an index list, it would be clearer to put this line just above `std::vector<double>` to group like types.